game-core/src/main/java/games/strategy/triplea/ai/pro/data/ProPurchaseOption.java

```diff
@@ -255,7 +255,8 @@ private double calculateLandDistanceFactor(final int enemyDistance) {
     final double distance = Math.max(0, enemyDistance - 1.5);
     final int moveValue = isLandTransport ? (movement + 1) : movement;
     // 1, 2, 2.5, 2.75, etc
-    final double moveFactor = 1 + 2 * (Math.pow(2, moveValue - 1) - 1) / Math.pow(2, moveValue - 1);
+    final double moveFactor =
+        1.0 + 2.0 * (Math.pow(2, moveValue - 1.0) - 1.0) / Math.pow(2, moveValue - 1.0);
```

Comment:
Haven't checked, but maybe when using `1d` and `2d` instead of `1.0` and `2.0` it might fit into a single line?
But not really important, similar thing like the style preference earlier.
It would be interesting to see if we consistently use one form over the other, or it's just a 50:50 chance each time

Reply:
It appears we have no examples of `1d` or `2d`, we have examples of `1.0d`
`1.0` AFAIK is a double, the `1.0d` should be unnecessary.

TBH I think there are other issues beyond trying to have this fit on one line. The numerator is seemingly magic, the addition and subtraction operations do not make sense off-the-bat, this takes a good bit of reverse engineering to understand. Focusing on formatting IMO is going to be the wrong focus for right now.